nrich-search/src/test/java/net/croz/nrich/search/repository/stub/TestEntitySearchRequest.java

```diff
@@ -71,6 +71,8 @@ public class TestEntitySearchRequest {
 
     private String nestedEntityNonExisting;
 
+    private String elementCollection;
```

Comment:
Maybe rename to `elementCollectionEntry`.

Reply:
Since it is matched by name that would require additional search configuration, this way I wanted to show basic and most used case of searching collection.

Reply:
Ok, we can leave it like this.